public/sass/elements/_reset.scss

```diff
@@ -117,3 +117,7 @@ td {
   line-height: inherit;
   font-weight: normal;
 }
+
+abbr[title], acronym[title] {
```

Comment:
I was thinking along the lines of http://jsbin.com/vejeqaxiwu/edit?html,css,output but I guess utilities would probably be forced anyway.